apps/examples/testcase/le_tc/filesystem/fs_main.c

```diff
@@ -1639,19 +1634,37 @@ static void tc_driver_mtd_config_ops(void)
 * @precondition     NA
 * @postcondition    NA
 */
-#if defined(CONFIG_MTD_FTL) && defined(CONFIG_BCH)
+#if defined(CONFIG_MTD_FTL) && defined(CONFIG_BCH) && defined(CONFIG_FLASH_PARTITION)
 static void tc_driver_mtd_ftl_ops(void)
 {
 	int fd;
 	int ret;
 	char *buf;
+	char *types = CONFIG_FLASH_PART_TYPE;
+	char MTD_FTL_PATH[DEVNAME_LEN];
+	int partno = 0;
 
 	buf = (char *)malloc(BUF_SIZE);
 	if (!buf) {
 		printf("Memory not allocated \n");
 		return;
 	}
 
+	memset(MTD_FTL_PATH, 0, DEVNAME_LEN);
+	while (*types) {
+		if (!strncmp(types, "smartfs,", 8)) {
```

Comment:
why smartfs? smartfs doesn't use FTL.
Actually you don't have to test ftl here, because there's test case for bch, and bch using ftl tho.